test/ns_tracker/core_test.clj

```diff
@@ -132,7 +132,7 @@
     (testing "self-dependencies throw an AssertionError"
       (spit (file "tmp/example/ns1.clj") '(ns example.ns1 (:require [example.ns1])))
       (Thread/sleep 1000)
-      (is (thrown? AssertionError (ns-tracker [(file "tmp")]))))
+      (is (thrown-with-msg? AssertionError #"self-referential dependency example.ns1" (ns-tracker [(file "tmp")]))))
```

Comment:
Can you ensure this line is under 80 characters?

Reply:
Sorry, missed that one, reformatted the line to be under 80 chars